core/src/main/resources/META-INF/services/io.questdb.griffin.FunctionFactory

```diff
@@ -509,6 +509,18 @@ io.questdb.griffin.engine.functions.groupby.LastLongGroupByFunctionFactory
 io.questdb.griffin.engine.functions.groupby.LastGeoHashGroupByFunctionFactory
 io.questdb.griffin.engine.functions.groupby.LastUuidGroupByFunctionFactory
 
```

Comment:
Looks like there's no first_not_null/last_not_null implementation for geohash or ipv4.

Reply:
Sorry that I currently don't have usage for these 2 data type and have excluded them in the submit.

Reply:
@bziobrowski what's the outcome of this one? Probably we should create a follow-up patch ourselves?

Reply:
Yes, I'll add them once this PR gets merged.

Reply:
Awesome. It's a :white_check_mark: then.